### DISCUSSION TITLE
How to upgrage the storage space of github pages

### DISCUSSION THREAD

**Discussion by impressstudio** [community/community#22655] [Other Feature Feedback, Questions, & Ideas]
<p>i want to know the details about storage space and limitations and upgradtion<br>
plz help me</p>

**Comment by tmetz** [comment, marked as answer]
<p>Hi there! :wave:</p>
<p>GitHub Pages is shared hosting, so there is no allocation of resources specific to an individual site.</p>
<p>To ensure good performance for our users, we recommend repositories be kept under 1GB each:</p>
<p><a href="https://docs.github.com/en/github/managing-large-files/what-is-my-disk-quota#file-and-repository-size-limitations" class="onebox" target="_blank">https://docs.github.com/en/github/managing-large-files/what-is-my-disk-quota#file-and-repository-size-limitations</a></p>
<p>There is only one plan, so there is no upgrade possible.</p>
<p>I hope this information helps!</p>

**Comment by impressstudio** [comment]
<p>if i exceed the storage limit</p>
<p>will u block me</p>
